i18n/en.json

```diff
@@ -2435,7 +2435,9 @@
   "apps.error.command.unknown_channel": "Unknown channel for field `{fieldName}`: `{option}`.",
   "apps.error.command.unknown_option": "Unknown option for field `{fieldName}`: `{option}`.",
   "apps.error.command.unknown_user": "Unknown user for field `{fieldName}`: `{option}`.",
-  "apps.error.form.no_call": "`call` is not defined.",
+  "apps.error.form.no_submit": "`submit` is not defined.",
+  "apps.error.form.no_source": "`source` is not defined.",
+  "apps.error.form.no_lookup": "`looukp` is not defined.",
```

Comment:
I've been wanting to get these debug strings out of i18n. Just made a meeting item here https://community-daily.mattermost.com/core/pl/hh6mfk6nqfyy8rbrp8u7x63wsw